onboarding-manager/internal/handlers/controller/reconcilers/host.go

```diff
@@ -260,11 +260,17 @@ func (hr *HostReconciler) invalidateHost(ctx context.Context, host *computev1.Ho
 	}
 
 	untrustedHost := computev1.HostResource{
-		ResourceId:          host.GetResourceId(),
-		CurrentState:        computev1.HostState_HOST_STATE_UNTRUSTED,
-		HostStatus:          om_status.AuthorizationStatusInvalidated.Status,
-		HostStatusIndicator: om_status.AuthorizationStatusInvalidated.StatusIndicator, // #nosec G115
-		HostStatusTimestamp: uint64(time.Now().Unix()),                                // #nosec G115
+		ResourceId:                  host.GetResourceId(),
+		CurrentState:                computev1.HostState_HOST_STATE_UNTRUSTED,
+		HostStatus:                  om_status.AuthorizationStatusInvalidated.Status,
+		HostStatusIndicator:         om_status.AuthorizationStatusInvalidated.StatusIndicator, // #nosec G115
+		HostStatusTimestamp:         uint64(time.Now().Unix()),                                // #nosec G115
```

Comment:
maybe for the ts - we dont really need. IIRC Inventory has an hook doing that for free on updates

Reply:
Removed